### ISSUE TITLE
Upgrading to 6.3.1 leads to incorrect newlineAfterBlock warnings

### ISSUE THREAD

**Issue by kaiyoma** [lesshint/lesshint#513]
**Which version of `lesshint` are you using?**
6.3.1

**How are you running `lesshint`? CLI, Node.js API, Grunt/Gulp plugin?**
CLI

**What's your `.lesshintrc` configuration?**

```js
{
  "excludedFiles": [
    "src/stylesheets/fonts/*.less"
  ],
  "idSelector": {
    "enabled": false
  },
  "qualifyingElement": {
    "allowWithClass": true
  },
  "spaceAroundComma": {
    "allowNewline": true
  },
  "universalSelector": {
    "enabled": false
  },
  "zeroUnit": {
    "style": "no_unit"
  }
}
```

**If you're reporting a bug, please show us some code that's failing.**

```less
.classname {
  .mixin-1;
  .mixin-2;
  style: property;
}
```

**What did you expect to happen?**
The linting to pass.

**What's the actual result?**
lesshint complains that the first mixin invocation should be followed by a newline, but this seems wrong to me. It's not a block, it's a mixin. Weirdly, it _doesn't_ complain about a missing newline after the second mixin invocation.

**Comment by kaiyoma** [comment]
Ah, I forgot to mention that I'm trying to upgrade from 5.3.0. No previous version of lesshint (that we've used on our project in the last 3 years) has ever complained about this code.

**Comment by jwilsson** [comment]
Thanks for the issue! I can reproduce it.

Looks like the `newlineAfterBlock` fix in `6.3.1` wasn't 100%, I'll take a look,

**Comment by jwilsson** [comment]
This should be fixed in `6.3.2`.

**Comment by kaiyoma** [comment]
Yup! Looking good over here. The new version is even finding a few errors in our code (that have been there for a while). 😄
